code/datums/movement_controller/pod.dm

```diff
@@ -80,7 +80,7 @@
 
 
 	update_owner_dir(var/atom/movable/ship) //after move, update ddir
-		owner.dir = last_dir
+		owner?.dir = last_dir
```

Comment:
this is valid because if owner is null, we're in the middle of disposing the holder anyways